src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InjectDirective.cs

```diff
@@ -18,6 +18,7 @@ public static class InjectDirective
             {
                 builder.AddTypeToken().AddMemberToken();
                 builder.Usage = DirectiveUsage.FileScopedMultipleOccurring;
+                builder.Description = "Inject a service into your document.";
```

Comment:
@DamianEdwards would appreciate if you could improve or "ok" on each of these descriptions. See the PR description for how these descriptions make it into the completion list.

Reply:
These should be resources YO.

Reply:
"Declare a property and inject a service from the application's service container into it."